duckduckgo/macos-browser
# Update autofill empty state pictograms

Task/Issue URL: https://app.asana.com/0/1203822806345703/1207526334057960/f


**Description**:

Passwords (as part of [✓ macOS: Make passwords easier to discover](https://app.asana.com/0/0/1206531758082882)) have an updated pictogram in use, with the new style of DDG across the board. Payment Methods and Identities still have the old DDG style.

This updates the remaining empty states of Autofill: Payment Methods and Identities.

**Steps to test this PR**:
1. Go to Debug -> Reset Data -> Reset Autofill Data
2. Go to (...) -> Passwords & Autofill
3. Select Identities from the picker
4. Check the empty state image has been updated
5. Select Credit Cards from the picker
6. Check the empty state image has been updated



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
